test/ci/buildkite/Dockerfile.ubuntu18.04

```diff
@@ -0,0 +1,54 @@
+
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+# Docker file to setup build envrionment for nGraph-TensorFlow
+
+FROM ubuntu:18.04
+
+# Setup the environment
+ENV https_proxy http://proxy-chain.intel.com:912
+ENV http_proxy http://proxy-chain.intel.com:911
+
+RUN apt-get update &&  apt-get install -y \
+    vim \
+    python \
+    python-pip \
+    python3-pip \
+    git \
+    unzip zip wget \
+    sudo \
+    zlib1g zlib1g-dev bash-completion \
+    build-essential cmake \
+    clang-format-3.9
+
+RUN pip install -U virtualenv==16.0.0
+
+#install latest cmake
+ADD https://cmake.org/files/v3.7/cmake-3.7.2-Linux-x86_64.sh /cmake-3.7.2-Linux-x86_64.sh
+RUN mkdir /opt/cmake
+RUN sh /cmake-3.7.2-Linux-x86_64.sh --prefix=/opt/cmake --skip-license
+RUN ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake
+RUN cmake --version
+
+#Install Bazel 
+ADD https://github.com/bazelbuild/bazel/releases/download/0.24.1/bazel-0.24.1-installer-linux-x86_64.sh ./bazel-0.24.1-installer-linux-x86_64.sh
```

Comment:
We need to document somewhere that that this file also needs to change when bazel version etc changes. Perhaps in the wiki (https://github.com/tensorflow/ngraph-bridge/wiki/Update-TF-version-in-ngraph-bridge), since we mostly refer to that for upgrade processes.